apps/forwarder-sriov/forwarder.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     metadata:
       labels:
         app: forwarder-sriov
+        "spiffe.io/spiffe-id": "true"
```

Comment:
Apply thisalso  for other places.
```suggestion
        spiffe.io/spiffe-id: true
```

Reply:
key `spiffe.io/spiffe-id` is not recognized without quotes

Reply:
ok, lets then update "true" to true

Reply:
ok, done